SDK/AppCenterData/Microsoft.AppCenter.Data/Microsoft.AppCenter.Data.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0;portable-net45+win8+wpa81+wp8</TargetFrameworks>
+    <Optimize>true</Optimize>
+    <Copyright>Microsoft Corp. All rights reserved.</Copyright>
+    <Product>Microsoft.AppCenter.Data</Product>
+    <Company>Microsoft Corporation</Company>
+    <Version>2.0.0-SNAPSHOT</Version>
+    <AssemblyVersion>0.0.0.0</AssemblyVersion>
+    <FileVersion>2.0.0.0</FileVersion>
+    <AssemblyTitle>Microsoft.AppCenter.Data</AssemblyTitle>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Microsoft.AppCenter.Data.xml</DocumentationFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.1.0" PrivateAssets="All" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Models\Page.cs" />
+    <Compile Remove="Models\PendingOperation.cs" />
+    <Compile Remove="Models\TokenResult.cs" />
+    <Compile Remove="Models\TokensResponse.cs" />
+    <Compile Remove="Models\PaginatedDocuments.cs" />
```

Comment:
what are those compile remove lines?

Reply:
VS make THIS for "Remove the file from a project"
🤦‍♂️